docs/topics/logging.txt

```diff
@@ -261,12 +265,37 @@ First, here's a simple configuration which writes all request logging from the
 If you use this example, be sure to change the ``'filename'`` path to a
 location that's writable by the user that's running the Django application.
 
-Second, here's an example of a fairly complex logging setup, configured using
+Second, here's an example of how to make the logging system print all of
+Django's logging to the console based on the value of an environment variable.
+It overrides the fact that ``django.request`` and ``django.security`` don't
+propagate their log entries. It may be useful during local development. Set the
+environment variable ``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging
+calls in the console::
+
+    import os
+
+    LOGGING = {
+        'version': 1,
+        'disable_existing_loggers': False,
+        'handlers': {
+            'console': {
+                'class': 'logging.StreamHandler',
+            },
+        },
+        'loggers': {
+            'django': {
+                'handlers': ['console'],
+                'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
+            },
+        },
+    }
+
+Finally, here's an example of a fairly complex logging setup, configured using
```

Comment:
I know this wording pre-existed this PR, but it seems strange to me to say "configured using `logging.config.dictConfig`" here. All of these examples use `dictConfig`, because that's what `LOGGING_CONFIG` does by default. This example doesn't use `dictConfig` any more than anything in this section. And it seems to confuse with the technique below of setting `LOGGING_CONFIG=None` and then using `dictConfig` yourself directly.

I think it should be said somewhere in this section (if it isn't already) that the default behavior is to pass `settings.LOGGING` to `dictConfig` (and link to the `dictConfig` docs), but I'm not sure why it should be said specifically for this example.